WordPress/Classes/Utility/Automated Transfer/AutomatedTransferHelper.swift

```diff
@@ -0,0 +1,330 @@
+        static let alertCancel = NSLocalizedString("Cancel", comment: "Cancel button.")
+        static let alertInstall = NSLocalizedString("Install", comment: "Confirmation button displayd in alert displayed when user installs their first plugin.")
+
+        // Eligibility errors.
+        static let eligibilityUnverifiedEmailError = NSLocalizedString("Plugin feature requires a verified email address.", comment: "Error displayed when trying to install a plugin on a site for the first time.")
+        static let eligibilityExcessiveUsageError = NSLocalizedString("Plugin cannot be installed due to disk space limitations.", comment: "Error displayed when trying to install a plugin on a site for the first time.")
+        static let eligibilityNoBusinessPlanError = NSLocalizedString("Plugin feature requires a business plan.", comment: "Error displayed when trying to install a plugin on a site for the first time.")
+        static let eligibilityVIPSitesError = NSLocalizedString("Plugin cannot be installed on VIP sites.", comment: "Error displayed when trying to install a plugin on a site for the first time.")
+        static let eligibilityNotAdminError = NSLocalizedString("Plugin feature requires admin privileges.", comment: "Error displayed when trying to install a plugin on a site for the first time.")
+        static let eligibilityNotDomainOwnerError = NSLocalizedString("Plugin feature requires primary domain subscription to be associated with this user.", comment: "Error displayed when trying to install a plugin on a site for the first time.")
+        static let eligibilityNoCustomDomainError = NSLocalizedString("Plugin feature requires a custom domain.", comment: "Error displayed when trying to install a plugin on a site for the first time.")
+        static let eligibilityGreyListedError = NSLocalizedString("Plugin feature requires the site to be in good standing.", comment: "Error displayed when trying to install a plugin on a site for the first time.")
+        static let eligibilityPrivateSiteError = NSLocalizedString("Plugin feature requires the site to be public.", comment: "Error displayed when trying to install a plugin on a site for the first time.")
+        static let eligibilityEligibilityGenericError = NSLocalizedString("Plugin feature is not available for this site.", comment: "Error displayed when trying to install a plugin on a site for the first time.")
+
+        // Other
+        static let refreshInterval: Int = 3
+        static let delaySequence = [Constants.refreshInterval]
+        static let maxRetries = 10
+        static let installAlmostDone = NSLocalizedString("We're doing the final setup — almost done…", comment: "Title of progress label displayed when a first plugin on a site is almost done installing.")
```

Comment:
I don't think we should use a em dash (`—`) with spaces around it—either no spaces, or a regular dash or en dash with spaces. I searched the codebase and we have many uses of a spaces en dash and also a few uses of a em dash with spaces and I'd just like to say that the inconsistency makes me a little bit sad.

Reply:
fixed!